torchvision/io/_video_opt.py

```diff
@@ -306,13 +306,13 @@ def _read_video_from_memory(
     video_height=0,  # type: int
```

Comment:
Can I switch this to Python 3 type annotations or will this mess with `torch.script`?

Reply:
we can use Python3 type annotations now